File: Python/payments/card.py
from datetime import date, datetime
from payment import payment

class card(payment):
    number  = int
    cvv     = int
    date    = str

    def __init__(self, id, number, cvv, date):
        super().__init__(id)
        self.number = number
        self.cvv    = cvv
        self.date   = date

File: Java/Payments/Card.java
package Java.Payments;

import java.util.Date;
import Java.Payment;

public class Card extends Payment {
    
    Integer number;
    Integer cvv;
    Date date;

    public Card(Integer id, Integer number, Integer cvv, Date date){
        super(id);
        this.number = number;
        this.cvv = cvv;
        this.date = date;
    }
}
